mlflow/store/model_registry/dbmodels/models.py

```diff
@@ -170,3 +170,29 @@ def __repr__(self):
     # entity mappers
     def to_mlflow_entity(self):
         return ModelVersionTag(self.key, self.value)
+
+
+class SqlRegisteredModelAlias(Base):
+    __tablename__ = "registered_model_aliases"
+    name = Column(String(256), ForeignKey("registered_models.name", onupdate="cascade"))
+    alias = Column(String(256), nullable=False)
+    version = Column(Integer, nullable=False)
+
+    # linked entities
+    registered_model = relationship(
+        "SqlRegisteredModel", backref=backref("registered_model_aliases", cascade="all")
+    )
+    model_version = relationship(
+        "SqlModelVersion",
+        foreign_keys=[name, version],
+        backref=backref("registered_model_aliases", cascade="all"),
+    )
+
+    __table_args__ = (
+        PrimaryKeyConstraint("name", "alias", name="model_version_alias_pk"),
+        ForeignKeyConstraint(
```

Comment:
Do we want this foreign key to make it easier to access aliases that belong to a particular version?